e2e/helpers/testUser.ts

```diff
@@ -45,6 +43,16 @@ export async function deleteTestUser(
   return testUserId
 }
 
+export async function cleanupTestUser(testUserId: string, userPage: Page) {
+  await deleteVerificationSubmissionIfExists(
+    testUserId,
+    getAdminPAT(),
+    userPage,
+  )
+  const result = await deleteTestUser(getEndpoint(), testUserId, getAdminPAT())
+  expect(result).toEqual(testUserId)
+}
```

Comment:
Remove associated verification submission if a user has requested verification or is verified, so the test user can be deleted